### PR TITLE
Update requirements versions

### PR DESCRIPTION
**Pull request by nvkevlu**
Updates the version of requirements to 2.5.0rc4.

### Description

Updates the version of requirements to 2.5.0rc4.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Review by chesterxgchen — DISMISSED** [review]

File: examples/advanced/brats18/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-nvflare~=2.4.0rc
+nvflare~=2.5.0rc4
```
Comment (YuanTingHsieh) [inline examples/advanced/brats18/requirements.txt:1]: Do we need the rc4 here? 
Or "rc" is enough?

Did you try if rc5 come, this expression will install rc5?

**Review by YuanTingHsieh — COMMENTED** [review]
One question, and please resolve the conflicts with some files that got changed by other PR thanks!

**Comment by nvkevlu** [comment]
Should we use 2.5.0rc4 or 2.5.0rc?

**Review by YuanTingHsieh — APPROVED** [review]
thanks, LGTM

**Pull request merged**